src/map/generateStops.js

```diff
@@ -207,7 +207,7 @@ export default function (layer, timefield, dispatch, nextIndex) {
       ? categories.color
       : (stops && stops[4]) || getColorBrewerColor(color, clusters) || color;
   const rawData =
-    layer.data ||
+  ((layer.data && layer.data.features) || layer.data) ||
```

Comment:
@kahummer Wouldn't this condition be always true regardless of whether `layer.data.feature` is defined or not

Reply:
the condition will pick layer.data.features as the raw data if that exists otherwise it'll pick layer.data